iai_boxy_base_2/robots/base.xacro

```diff
@@ -24,6 +24,106 @@
 				</geometry>
 			</collision>
 		</link>
+		
+		<xacro:if value="${with_wheels}">
+			<joint name="wheel_left_up_joint" type="revolute">
+				<limit lower="-1000.0" upper="1000.0" effort="200.0" velocity="1.0" />
+				<origin xyz="0.13 0.09 -0.06" rpy="0.0 0.0 0.0" />
+				<parent link="base_link" />
+				<child link="wheel_left_up" />
+				<axis xyz="0 1 0" />
+			</joint>
+			<link name="wheel_left_up">

```

Comment:
i think better names would be `wheel_front_left` and so on. up and down don't really make sense.

Reply:
the wheels also appear red in rviz. Can you color them grey?

Reply:
the new wheels overlap with the old ones, that are part of the base_link. if the point is, to make donbot pretty, then the base link mesh needs to be cut into pieces

Reply:
the collision mesh should just be a simple cylinder with the same size as the wheel, to speed up collision checking